package.json

```
@@ -3,8 +3,11 @@
   "version": "0.0.0",
   "description": "A small app to help scripts access private resources on Solid Pods",
   "main": "dist/index.js",
+  "bin": {
+    "generate-oidc-token": "cli/cli.sh"
+  },
```

Knowing ourselves, might be best to keep this stable against future package renames :P

```suggestion
  "bin": "cli/cli.sh",
```

---

Done in 9b35c7f0f6fe756ac6d5b43684683b8ee63826a2. I'll see if that doesn't confuse `npx` though :)

---

All clear, I learned something today :)